pkg/comp-functions/functions/common/utils.go

```diff
@@ -0,0 +1,60 @@
+package common
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/vshn/appcat/v4/pkg/common/quotas"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/runtime"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+type plans map[string]plan
+
+type plan struct {
```

Comment:
Don't we have already these plans somewhere defined? This seems like a VSHN service struct, not specific to Minio.

Reply:
Yes, they are already defined in `pkg/common/quotas/plans.go` but the type itself is currently private. So they need to be made public.

Reply:
Refactored the plans/resources a bit. Please check again.